css-typed-om/Overview.bs

```diff
@@ -1997,16 +1997,11 @@ is a [=list=] of {{CSSTransformComponent}}s.
 
     2. [=list/For each=] |func| in |this|’s [=values to iterate over=]:
 
-        1. Let |funcMatrix| be |func|’s equivalent 4x4 transform matrix,
-            as defined in [[css-transforms-1#mathematical-description]].
-
-            As the entries of such a matrix are defined relative to the ''px'' unit,
-            if any <<length>>s in |func| involved in generating the matrix
-            are not [=compatible units=] with ''px'',
-            [=throw=] a {{TypeError}}.
+        1. Let |funcMatrix| be the {{DOMMatrix}} returned by
+            calling {{CSSTransformComponent/toMatrix()}} on |func|.
         2. Set |matrix| to the result of multiplying |matrix|
-            and |funcMatrix|.
-        3. If |func|’s {{CSSTransformComponent/is2D}} internal slot is `false`,
+            and the matrix represented by |funcMatrix|.
```

Comment:
Can I just multiply two `DOMMatrix` together and will it correctly handle `is2D`?